modules/fieldlinessequence/util/fieldlinesstate.cpp

```diff
@@ -296,6 +296,8 @@ void FieldlinesState::saveStateToOsfls(const std::string& absPath) {
         ofs.write(reinterpret_cast<char*>(vec.data()), sizeof(float) * nPoints);
     }
     ofs.write(allExtraQuantityNamesInOne.c_str(), nStringBytes);
+
+    ofs.close();
```

Comment:
No need to explicitly close the file here as it automatically gets closed at the end of the scope since it is a local variable